app/queries/best_sellers.rb

```diff
@@ -0,0 +1,13 @@
+class BestSellers < Rectify::Query
+  def query
+    Book.includes(:authors).find(best_sellers_ids)
```

Comment:
Query should return a relation instead of array. Consider using `where(id: best_sellers_ids)` or something else

Reply:
fixed